script.js

```diff
@@ -166,25 +239,153 @@ const sideMenu = {
       scroll: false,
       borderless: true,
       css: "side-bar",
+      on: {
+        onAfterSelect: function(id) {
+          if (id == "users") {
+            changeListColors();
+          }
+          $$(id).show();
+        }
+      },
       data: [
-        { id: 1, title: "Dashboard" },
-        { id: 2, title: "Users" },
-        { id: 3, title: "Products" },
-        { id: 4, title: "Locations" }
+        { id: "dashboard", title: "Dashboard" },
+        { id: "users", title: "Users" },
+        { id: "products", title: "Products" },
+        { id: "locations", title: "Locations" }
       ]
     },
     {},
     {
       maxHeight: 40,
       borderless: false,
       css: "connection-status",
-      template: "<span class=' webix_icon wxi-check'>Connected</span>"
+      template: "<span class='webix_icon wxi-check'>Connected</span>"
     }
   ]
 };
 
+const filterUsersList = () => {
+  const value = $$("listFilter")
+    .getValue()
+    .toLowerCase();
+  $$("usersList").filter(function(obj) {
+    return (
+      obj.name.toLowerCase().indexOf(value) !== -1 ||
+      obj.country.toLowerCase().indexOf(value) !== -1
+    );
+  });
+};
+
+const randomColor = () => {
+  let colorArray = [];
+  for (let i = 0; i < 3; i++) {
+    colorArray.push(Math.floor(Math.random() * (255 - 0) + 0));
+  }
+  return `rgb(${colorArray.join(",")}, 0.2)`;
+};
+
+const usersList = {
+  rows: [
+    {
+      view: "toolbar",
+      elements: [
+        {
+          view: "text",
+          id: "listFilter",
+          on: { onTimedKeyPress: filterUsersList }
+        },
+        {
+          view: "button",
+          label: "Sort asc",
+          gravity: 0.2,
+          css: "webix_primary",
+          click: () => {
+            $$("usersList").sort("name", "asc");
+          }
+        },
+        {
+          view: "button",
+          label: "Sort Desc",
+          gravity: 0.2,
+          css: "webix_primary",
+          click: () => {
+            $$("usersList").sort("name", "desc");
+          }
+        }
+      ]
+    },
+    {
+      view: "list",
+      id: "usersList",
+      template:
+        "<div class= 'users-list--flex'><span>#name# from #country#</span>{common.deleteIcon}</div>",
+      type: {
+        // userInfo: function(obj) {
+        //   `<span>${obj.name} from ${obj.country}</span>`;
+        // },
```

Comment:
Dont commit commented lines. Just remove it. If you need it in future you can check all your code in git.